### ISSUE TITLE
Error uploading after copy command

### ISSUE THREAD

**Issue by claasd** [conan-io/conan#943]
Hi,

after upgrading to conan 0.19 this morning, one step that worked with conan 0.18.1 now fails. 
I do the following (promoting tested packages to a public user): 
```bash
conan.exe install --update "MyPackage/1.8.0-alpha-build3@tmp/unstable" --all 
conan.exe copy "MyPackage/1.8.0-alpha-build3@tmp/unstable" public/unstable 
conan.exe upload "MyPackage/1.8.0-alpha-build3@public/unstable" --all --confirm -r myRemote --retry 3 
```
The new error message is:
```bash
ERROR: Trying to upload a package recipe without sources, and the remote for the sources no longer exists
```
Is there a new option I am missing?

Best,
Claas

**Comment by claasd** [comment]
Just an info: it works when I have the created packages available, just not when I need to install them from a remote first.

**Comment by memsharded** [comment]
No, it seems a bug from the new "exports_sources" feature. Having a look asap.

**Comment by memsharded** [comment]
Proposed a fix in PR #944, tentatively for a minor release, if you want to try it from sources, you could try my branch: https://github.com/memsharded/conan/tree/release/0.19.1

**Comment by claasd** [comment]
thanks for the quick fix. Any idea about the release date of 0.19.1?

**Comment by memsharded** [comment]
Have you already tried from sources? It is passing CI, a bit slow travis for OSX... We are waiting for it, and for further feedback in the meantime. We try to release minors asap, so we might try to release it today.

**Comment by memsharded** [comment]
Released in 0.19.1. Thanks for reporting!
